Sources/ParseSwift/Protocols/ParseObjectMutable.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ParseObjectMutable.swift
+//  ParseSwift
+//
+//  Created by Damian Van de Kauter on 30/10/2021.
+//  Copyright © 2021 Parse Community. All rights reserved.
+//
+
+import Foundation
+
+/**
+ The **ParseObjectMutable** protocol adds an empty version of the respective object. This can be used when you only need to update a
```

Comment:
```suggestion
 The `ParseObjectMutable` protocol adds an empty version of the respective object. This can be used when you only need to update a
```

Reply:
This change should be made to all names of types in the documentation. The change makes "links" in the documentation as opposed to making text bold

Reply:
Ok, will fix this. Good to know.